packaging/microshift/config.yaml

```diff
@@ -1,31 +1,18 @@
-dns:
-  # Base domain of the cluster. All managed DNS records will be sub-domains of this base.
-  #baseDomain: microshift.example.com
-
-network:
-  clusterNetwork:
-  # IP range for use by the cluster
-  #- cidr: 10.42.0.0/16
-
-  serviceNetwork:
-  # IP range for services in the cluster
-  #- 10.43.0.0/16
-
-  # Node ports allowed for services
-  #serviceNodePortRange: 30000-32767
-
-node:
-  # If non-empty, use this string to identify the node instead of the hostname
-  #hostnameOverride: ''
-
-  # IP address of the node, passed to the kubelet.
-  # If not specified, kubelet will use the node's default IP address.
-  #nodeIP: ''
-
-apiServer:
-  # The Subject Alternative Names for the external certificates in API server (defaults to hostname -A)
-  #subjectAltNames: []
-
-debugging:
-  # Log verbosity ('Normal', 'Debug', 'Trace', 'TraceAll'):
-  #logLevel: 'Normal'
+# See https://github.com/openshift/microshift/blob/main/docs/howto_config.md#configuration
```

Comment:
I'm worried that if we link to GitHub in the example files then we have to maintain those URLs for the product lifetime. Can we either link to product docs, or drop the links entirely and assume that users will search for the docs themselves?